scripts/sync.py

```diff
@@ -131,7 +131,10 @@ def do(args):
 
     # initialize (overwrite) metadata.csv using either REDCap or RPMS database
     if 'redcap' in args.input_sources or 'rpms' in args.input_sources:
-        lochness.initialize_metadata(Lochness, args)
```

Comment:
In PR #35 , all I had to do is bring this line back with an additional `multiple_site_in_a_repo=False` argument.